bin/install-pr

```diff
@@ -87,15 +95,16 @@ echo "### Loading images into Docker ###"
 image=$(docker load -i image-archives/cli-bin.tar | sed 's/Loaded image: //')
 tag=$(echo "$image" | cut -f 2 -d ":")
 
-if [ $is_kind ]
+if [ $is_kind ] || [ $is_k3d ]
 then
-  # KinD context strings are created by prepending `kind-` to the cluster name
-  if [ "$context" ]
+  distro=""
+  if [ $is_k3d ]
   then
-    name=$(echo "$context" | sed -n -E "s/(kind)-(.*)/\2/p")
+    distro="k3d"
+  else
+    distro="kind"
```

Comment:
tioli nit: You could do go-style here and save a couple of lines by initializing `distro="k3d"` and overriding only `if [$is_kind]`. Also this sets the default to `k3d` instead of `kind` which I believe is what most of us are using nowadays.

Reply:
Yep sounds good! `k3d` will be the "default" but we'll still need to pass the `--k3d` flag. The actual default of not passing anything is still just using an actual k8s cluster. I didn't plan to change that as part of this but maybe something to do if it seems necessary in the future.